[CI] Switch to check against expected result files for cpu inductor integration tests

desertfire opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #113668
* #113559
* #113446
* #113575
* #113574

Summary: With this, we can completely remove CI_SKIP from common.py.

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @chenyang78 @aakhundov @kadeng

desertfire committed:
[CI] Remove CI skip list for cpu inductor integration tests

Summary: With this, we can completely remove CI_SKIP from common.py.

[ghstack-poisoned]

desertfire committed:
Update on "[CI] Switch to check against expected result files for cpu inductor tests"


Summary: With this, we can completely remove CI_SKIP from common.py.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

desertfire commented on benchmarks/dynamo/ci_expected_accuracy/cpu_inductor_huggingface_inference.csv:
```diff
@@ -0,0 +1,185 @@
+name,accuracy,graph_breaks
```
Are you referring to the 3 lines of spaces? @jansel added a call to `tools/linter/adapters/no_merge_conflict_csv_linter.py` to format it in https://github.com/pytorch/pytorch/pull/112655/, which seems intended to prevent merge conflicts (https://github.com/pytorch/pytorch/blob/f9ea697112d4b6f2fef68569c875cae9b0136598/tools/linter/adapters/no_merge_conflict_csv_linter.py#L69C8-L69C8).

huydhn replied:
Yes, this is indeed to reduce merge conflicts when people updates the expected value for different models

desertfire committed:
Update on "[CI] Switch to check against expected result files for cpu inductor tests"


Summary: With this, we can completely remove CI_SKIP from common.py.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

desertfire committed:
Update on "[CI] Switch to check against expected result files for cpu inductor tests"


Summary: With this, we can completely remove CI_SKIP from common.py.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

desertfire committed:
Update on "[CI] Switch to check against expected result files for cpu inductor tests"


Summary: With this, we can completely remove CI_SKIP from common.py.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

desertfire committed:
Update on "[CI] Switch to check against expected result files for cpu inductor integration tests"


Summary: With this, we can completely remove CI_SKIP from common.py.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

desertfire committed:
Update on "[CI] Switch to check against expected result files for cpu inductor integration tests"


Summary: With this, we can completely remove CI_SKIP from common.py.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

desertfire committed:
Update on "[CI] Switch to check against expected result files for cpu inductor integration tests"


Summary: With this, we can completely remove CI_SKIP from common.py.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

desertfire committed:
Update on "[CI] Switch to check against expected result files for cpu inductor integration tests"


Summary: With this, we can completely remove CI_SKIP from common.py.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

desertfire committed:
Update on "[CI] Switch to check against expected result files for cpu inductor integration tests"


Summary: With this, we can completely remove CI_SKIP from common.py.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

desertfire committed:
Update on "[CI] Switch to check against expected result files for cpu inductor integration tests"


Summary: With this, we can completely remove CI_SKIP from common.py.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

desertfire committed:
Update on "[CI] Switch to check against expected result files for cpu inductor integration tests"


Summary: With this, we can completely remove CI_SKIP from common.py.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

desertfire commented:
@pytorchbot merge -f "touches inductor integration tests only and they have passed"
